src/Language/PureScript/Scheme/CodeGen/Transpiler.hs

```diff
@@ -88,16 +88,17 @@ caseAlternativesToScheme values caseAlternatives =
     condClauses = map makeClause alternatives
 
     makeClause (_valuesAndBinders, Left _xs) = error "Not implemented"
-    makeClause (valuesAndBinders, Right expr) = 
-      (makeTest valuesAndBinders, exprToScheme expr)
+    makeClause (valuesAndBinders, Right result) = 
+      (makeTest valuesAndBinders,
+       valuesAndBindersAndResultToLet valuesAndBinders result)
```

Comment:
The first element of the tuple is the test and the second argument is the result.